Source/WebKit/Platform/spi/Cocoa/CorePredictionSPI.h

```diff
@@ -47,6 +47,7 @@ struct svm_node
 extern "C" {
 #endif
 
+struct svm_model;
```

Comment:
The other code in this file seems to be striving for C compatibility. So this would be:
```suggestion
typedef struct svm_model svm_model;
```
Or maybe we don't need that style any more because of changes to the C language over time or because this is only included in C++ files now.